extensions/cornerstone/src/components/WindowLevelActionMenu/WindowLevelActionMenu.tsx

```diff
@@ -59,6 +99,94 @@ export function WindowLevelActionMenu({
     [commandsManager]
   );
 
+  const onSetColorLUT = useCallback(
+    props => {
+      const immediate = true;
+      commandsManager.run({
+        commandName: 'setViewportColormap',
+        commandOptions: {
+          ...props,
+          immediate,
+        },
+        context: 'CORNERSTONE',
+      });
+    },
+    [commandsManager]
+  );
+
+  const onSetColorbar = useCallback(
+    props => {
+      const currentColorMap = getCurrentViewportColormap(viewportId);
```

Comment:
what is current about this function since it is getting viewportId, how about `getViewportColormap`?

Reply:
done